metadata.rb

```diff
@@ -8,8 +8,8 @@
 version           '7.1.3'
 source_url        'https://github.com/sous-chefs/postgresql'
 issues_url        'https://github.com/sous-chefs/postgresql/issues'
-chef_version      '>= 13.8'
+chef_version      '>=12.20'
```

Comment:
Please can this be reverted back to supporting chef 13
```suggestion
chef_version      '>=13.8'
```